src/ui/content/movies.tsx

```diff
@@ -14,40 +14,44 @@ interface Movie {
     title: string
 }
 
+interface lmProps {
+  searchString: string
+}
+
+function ListMovies(props: lmProps) {
+  const search = props.searchString
+  
+  const { loading:isLoading, error, data } = useQuery(MOV_Q, { variables: { search: search } })
+    
+  if (isLoading) return <p>Loading ...</p>
+  if (error) return <h1>Log in to view movies.</h1>
+
+  if (data.movies.length === 0) return <h1>No movies found.</h1>
+  const movies = data.movies
+  const listItems = movies.map((movie: Movie, index: number) =>
+        <h1 key={index}>{movie.title}</h1>
+  )
+  return listItems
+}
+
 //is it good to double check whether user is logged in? eg: in greeting, nav, content
 export function Movies() {
-    const [search, upSearch] = useState('')
-    const { loading, error, data } = useQuery(MOV_Q)
-    /*const [loadGreeting, { called, loading2, data2 }] = useLazyQuery(
-      MOV_Q,
-      { variables: { search: search } }
-    )*/
-    if (loading) return <p>Loading ...</p>
-    if(error) console.log(error)
-    if (error) return <h1>Log in to view movies.</h1>
-    if (data.movies.length === 0) return <h1>No movies found.</h1>
-    const movies = data.movies
-    console.log(movies)
-    const listItems = movies.map((movie: Movie, index: number) =>
-        <h1 key={index}>{movie.title}</h1>
-    )
-    
+    const [search, upSearch] = useState('')    
 
     function handleSubmit(event: any) {
```

Comment:
react has types for those event parameters:
```suggestion
    function handleSubmit(event: React.FormEvent) {
```